python/tests/test_lowlevel.py

```diff
@@ -1,13 +1,35 @@
 from __future__ import print_function
 import pytest
-from deltachat import capi, Account, const
+from deltachat import capi, Account, const, set_context_callback, clear_context_callback
+from deltachat.capi import ffi
+from deltachat.account import EventLogger
 
 
 def test_empty_context():
     ctx = capi.lib.dc_context_new(capi.ffi.NULL, capi.ffi.NULL, capi.ffi.NULL)
     capi.lib.dc_close(ctx)
 
 
+def test_callback_None2int():
+    ctx = capi.lib.dc_context_new(capi.lib.py_dc_callback, ffi.NULL, ffi.NULL)
+    set_context_callback(ctx, lambda *args: None)
+    capi.lib.dc_close(ctx)
+    clear_context_callback(ctx)
+
+
+def test_dc_close_events():
+    ctx = capi.lib.dc_context_new(capi.lib.py_dc_callback, ffi.NULL, ffi.NULL)
+    evlog = EventLogger(ctx)
+    evlog.set_timeout(5)
+    set_context_callback(ctx, lambda ctx, evt_name, data1, data2: evlog(evt_name, data1, data2))
+    capi.lib.dc_close(ctx)
+    # test that we get events from dc_close
+    print(evlog.get_matching("DC_EVENT_INFO", check_error=False))
```

Comment:
any chance we can check for the actual event thing, not just that any `info` was printed?